packages/date-picker/test/common.js

```diff
@@ -134,3 +134,17 @@ export function getFocusedCell(overlayContent) {
 
   return focusedCell;
 }
+
+/**
+ * Waits for the scroll to finish in the date-picker overlay content.
+ *
+ * @param {HTMLElement} overlayContent
+ */
+export async function waitForScrollToFinish(overlayContent) {
+  if (overlayContent._targetPosition) {
```

Comment:
**note:** Didn't find any better way of determining whether the overlay content is scrolling. Alternatively, it could check for the `scroll` events, but it would lead to a much more complicated solution.

Reply:
Seems reasonable. Introducing a private API like `isScrolling`, and adding tests for that would also be an option, though adding APIs just for tests is not ideal as well.